agent/internal/docker_credential_store.go

```diff
@@ -21,6 +21,14 @@ type credentialStore struct {
 	store    hclient.ProgramFunc
 }
 
+func getDockerConfigPath() (string, error) {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return homeDir, errors.Wrap(err, "unable to find user's HOME directory")
+	}
+	return path.Join(homeDir, ".docker/config.json"), nil
```

Comment:
non-blocking: I don't know what this would do on windows (but my guess is not work?) and we don't support it but maybe one less thing if we ever do.
```suggestion
	return path.Join(homeDir, ".docker", "config.json"), nil
```

Reply:
Good idea!